.gitignore

```diff
@@ -151,3 +151,4 @@ venv.bak/
 ### AppEngine template
 # Google App Engine generated folder
 appengine-generated/
+node_modules/
```

Comment:
ah, mine is ignored globally. I should have thought of people other than myself ;)